README.md

```
@@ -8,6 +8,47 @@ version](https://www.r-pkg.org/badges/version/ggExtra)](https://cran.r-project.o
 > the MIT license.*
 
 `ggExtra` is a collection of functions and layers to enhance ggplot2.
+## New feature: ggMarginalSignif()
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)